### PR TITLE
feat: connect tasks from db to redux store

### LINKED ISSUE #32: GRAPHQL:Fetch all Tasks given a Group ID

### PR DESCRIPTION
**Pull request by dffattal**
Closes #32

### CODE REVIEW

**Review by miggitymack — DISMISSED** [review]
Returns groups' tasks. Replace null with "No tasks"

File: app/redux/reducers/tasks.js [hunk 1]
```diff
@@ -32,10 +37,24 @@ const setBounty = setBounty => ({
...
 export const addSelectedTask = selectedTask => {
   return setSelectedTask(selectedTask);
 };
 
 export const addBounty = addBounty => {
   return setBounty(addBounty);
 };
+
+export const fetchGroupTasks = groupId => dispatch => {
+  fetch(`http://192.168.2.8:4000/?query=query%7Bgroups(id%3A%20${groupId})%7Btasks%7Bid%20description%7D%7D%7D%0A`)
```
Comment (JeffKandel) [inline app/redux/reducers/tasks.js]: I think we'll want to abstract this out.  But not quite yet.

**Review by JeffKandel — DISMISSED** [review]
Looks good! Merge away!

**Review by JeffKandel — DISMISSED** [review]
Please merge.

**Review by JeffKandel — APPROVED** [review]
I approve of this change.  I am going to have you abstract the query in the next branch though.  the Groups reducer as an example after branch 31 has been merged.

**Pull request merged**